karate-tests/src/test/java/mocks/sandbox/get-patient-search.js

```diff
@@ -1,119 +1,122 @@
 /*
     Supporting functions for building responses
 */
-function janeSmithSearchsetWithScore(score) {
-    return {
-        "resourceType": "Bundle",
-        "type": "searchset",
-        "total": 1,
-        "entry": [
-            {
-                "fullUrl": "https://api.service.nhs.uk/personal-demographics/FHIR/R4/Patient/9000000009",
-                "search": {"score": score},
-                "resource": SEARCH_PATIENT_9000000009
-            }
-        ]
-    }
+
+/* Karate objects */
+/* global request, response */
```

Comment:
Are these comments still required?

Reply:
Yes these comments are intentionally here - they're commands for esint to understand that these are globals